dds_web/commands.py

```diff
@@ -336,3 +343,63 @@ def lost_files_s3_db(action_type: str):
 
     else:
         flask.current_app.logger.info("Found no lost files")
+
+
+@click.command("monitor-usage")
+@flask.cli.with_appcontext
+def monitor_usage():
+    """Check the units storage usage and compare with chosen quota."""
+    flask.current_app.logger.info("Starting: Checking unit quotas and usage...")
+
+    # Imports
+    # Own
+    from dds_web.database import models
+    import dds_web.utils
+
+    # Email settings
+    recipient: str = flask.current_app.config.get("MAIL_DDS")
+    default_subject: str = "DDS: Usage quota warning!"
+
+    # Run task
+    for unit in models.Unit.query:
+        flask.current_app.logger.info(f"Checking quotas and usage for: {unit.name}")
+
+        # Get info from database
+        quota: int = unit.quota
+        warn_after: int = unit.warning_level
+        current_usage: int = unit.size
+
+        # Check if 0 and then skip the next steps
+        if not current_usage:
+            flask.current_app.logger.info(
+                f"{unit.name} usage: {current_usage} bytes. Skipping percentage calculation."
+            )
+            continue
+
+        # Calculate percentage of quota
+        perc_used = current_usage / quota
+
+        # Information to log and potentially send
+        info_string: str = (
+            f"- Quota:{quota} bytes\n"
+            f"- Warning level: {warn_after*quota} bytes ({warn_after}%)\n"
```

Comment:
```
- Quota:1000000000 bytes
- Warning level: 800000000.0 bytes (0.8%)
- Current usage: 439221575 bytes (0.439221575%)
```
the figures in the parentesis are not actually printed as percentages, we can multiply by 100 and round the "Current usage"